rust/numaflow-core/src/sink.rs

```diff
@@ -489,17 +486,6 @@ impl SinkWriter {
                             .get_or_create(pipeline_forward_metric_labels("Sink"))
                             .inc_by((total_msgs - total_valid_msgs) as u64);
                     }
-
-                    processed_msgs_count += total_msgs;
-                    if last_logged_at.elapsed().as_millis() >= 1000 {
-                        info!(
-                            "Processed {} messages at {:?}",
-                            processed_msgs_count,
-                            time::Instant::now()
-                        );
-                        processed_msgs_count = 0;
-                        last_logged_at = std::time::Instant::now();
-                    }
```

Comment:
where in sink are we printing this now? i see that in source we have it

Reply:
If source has it, we don't need to print in sink.